CMakeLists.txt

```
@@ -354,6 +375,34 @@ if(GUI)
             ${THREADING_LINK_LIBS}
   )
 
+  # Main Dissolve GUI
+  target_link_libraries(
+    ${qmlgui_target_name}
+    PUBLIC ${WHOLE_ARCHIVE_FLAG}
+            ${BASIC_LINK_LIBS}
+            # Module gui libs
+            ${MODULEGUI_LINK_LIBS}
+            moduleWidgetProducer
+            # Main gui libs
+            render
+            widgets
+            keywordWidgets
+            delegates
+            models
+            gui
```

```suggestion
            # Main gui libs
            gui-qml
```

Only need to link against out new gui-qml library.  It includes the inner libraries that we need (i.e. `models`).